src/solver/simulation/sim_calcul_economique.cpp

```diff
@@ -83,7 +83,7 @@ void SIM_InitialisationProblemeHebdo(Data::Study& study,
 
     auto& parameters = study.parameters;
 
-    problem.Expansion = parameters.expansion;
+    problem.Expansion = (parameters.mode == Data::stdmExpansion);
```

Comment:
`Data::stdmExpansion ` : **stdm** probably means **study mode**. 
`Data::SimExpansion ` would be better.

Reply:
Done ! See https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1771/commits/f63d6c7c37bd07c22f8c04559c7eebfd8cc12d79